lib/terraform/runner.rb

```diff
@@ -0,0 +1,193 @@
+require 'faraday'
+require 'tempfile'
+require 'zip'
+require 'base64'
+
+module Terraform
+  class Runner
+    class << self
+      def available?
+        return @available if defined?(@available)
+
+        response = terraform_runner_client.get('api/terraformjobs/count')
```

Comment:
Super-tiny-minor, but do we really need a list of all jobs to show availability?  That is, is there
a) a simpler query like maybe just hitting /api itself?
b) can we use a HEAD request?

Reply:
how about ping?

```ruby
irb(main):003:0> Terraform::Runner.send(:terraform_runner_client).get('api/ping').status
=> 200
```

Reply:
opened https://github.com/ManageIQ/manageiq-providers-embedded_terraform/pull/14 for this